demo/src/constrained_planning_demo.cpp

```diff
@@ -101,16 +101,16 @@ int main(int argc, char** argv)
 
   // Now wait for the user to press Next before trying the planar constraints.
   moveit_visual_tools.prompt(
-      "Press 'Next' in the RvizVisualToolsGui window to continue to the planar constraint example");
+      "Press 'Next' in the RvizVisualToolsGui window to continue to the orientation constraint example");
 
   // Clear the path constraints and markers for the next example
-  reset_demo();
+  /*reset_demo();*/
 
   // In the second problem we plan with the end-effector constrained to a plane.
   // We need to create a pose goal that lies in this plane.
   // The plane is tilted by 45 degrees, so moving an equal amount in the y and z direction should be ok.
   // Any goal or start state should also satisfy the path constraints.
-  target_pose = get_relative_pose(0.0, 0.3, -0.3);
+  /*target_pose = get_relative_pose(0.0, 0.3, -0.3);*/
```

Comment:
Remove?

Reply:
Just talked to Aditya and I think we should keep and uncomment as these are the move-on-line and move-within-plane demos

Reply:
Sounds good. I do suspect that the rotated bounding box constraints may not work with the current implementation of position constraints -- there is another overload that accepts a rigid transform from a coordinate frame `A` to `A_bar` which enables rotating the reference frame, and we may need to achieve this.

Trying it out will tell.